frontend/Views/Widgets/SummaryWidget/SummaryWidget.svelte

```diff
@@ -39,11 +43,36 @@
     });
     $: if (versions.length > 0) fetchRunResults(currentVersionIndex);
 
+    /**
+     * Assigns a name to each test if it is null and sorts the test info by test name.
+     *
+     * @param data - The test info data to process and sort.
+     * @returns The processed and sorted test info data.
+     */
+    function assignAndSortTestInfo(data: Record<string, TestInfo>): Record<string, TestInfo> {
+
+        Object.entries(data).forEach(([key, value]) => {
+            if (value.name === null) {
+                const parts = value.build_id.split("/");
+                value.name = parts[parts.length - 1];
+            }
+        });
+
+        const sortedEntries = Object.entries(data).sort(([, a], [, b]) => {
+            if (a.name && b.name) {
+                return a.name.localeCompare(b.name);
```

Comment:
nice, I can write test names in hebrew :)

Reply:
currently it takes mostly from build_id - I didn't see yet any `test_name` to be specified in `sct_test_run` table... 
You need use hebrew in jenkins thou :stuck_out_tongue: